### PR TITLE
Doctor api

### PR DESCRIPTION
**Pull request by kosher9**
Changes made in this pull request : 
- [x] Added API for doctors' general list.
- [x] Added API for doctors list created by a given user.
- [x] Removed database.yml file from tracking.

### CODE REVIEW

File: app/controllers/v1/doctors_controller.rb [hunk 1]
```diff
@@ -17,12 +18,16 @@ def create
   def destroy
     @doctor = Doctor.find(params[:id])
     if @doctor.destroy
-      render json: { status: 201, message: 'doctor has deleted successfully!', content: { doctor: @doctor } }
+      render json: { status: 201, message: 'doctor has been deleted successfully!', content: { doctor: @doctor } }
```
Comment (samiullahbahadur) [inline app/controllers/v1/doctors_controller.rb:21]: line 20 and line 21 is the same please fix it

File: app/controllers/v1/doctors_controller.rb [hunk 2]
```diff
@@ -17,12 +18,16 @@ def create
...
     else
       render json: { error: 401, message: ' Operation did not succeed!' }
     end
   end
 
+  def doctors_list
+    render json: Doctor.all
+  end
+
```
Comment (samiullahbahadur) [inline app/controllers/v1/doctors_controller.rb:30]: doctor_list action is same to the index action (render json: Doctor.all)

**Review by samiullahbahadur — CHANGES_REQUESTED** [review]
### **Changes Required**
Hi @kosher9 you did a good job so far.
your work need some few changes to get approved

File: app/controllers/v1/doctors_controller.rb (hunk 1 above)
Comment (kosher9) [reply app/controllers/v1/doctors_controller.rb:21]: Hey @samiullahbahadur, they are not the same, line 20 has been changed. That's why it is red.

Comment (kosher9) [reply app/controllers/v1/doctors_controller.rb:21]: Check it well please, they look like they are the same but they are quite different.

File: app/controllers/v1/doctors_controller.rb (hunk 2 above)
Comment (kosher9) [reply app/controllers/v1/doctors_controller.rb:30]: Check it well please, they look like they are the same but they are quite different.

**Review by samiullahbahadur — APPROVED** [review]
Approved
Hi @kosher9  in my opinion you project is completed the time to merge it

**Pull request merged**